.github/workflows/vast.yaml

```diff
@@ -69,8 +69,8 @@ jobs:
           apt-get -qqy -t buster-backports install libspdlog-dev
 
           # clang 9 (c.f. https://apt.llvm.org/)
+          curl -fsSL https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add -
```

Comment:
This now deviates from the link above, so please add a comment why that is. I know now, but the future maintainer of this file may not know from memory.

Reply:
The link from the previous discussion is not contained in the file anymore.

Reply:
Yes, but the code we run is not at https://apt.llvm.org, so please update that comment.

Reply:
I don't understand your comment. The code is right on the page you linked, except that we use `cURL` instead of `wget` as they do.
> To retrieve the archive signature:
> wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key|sudo apt-key add -
> # Fingerprint: 6084 F3CF 814B 57C1 CF12 EFD5 15CF 4D18 AF4F 7421

Reply:
That caused issues, right? Please add a comment that explains what those issues were so we don't run into them again in the future.

Reply:
No, using `wget` did not cause any issues. We used to call `apt-key adv <keyserver>` directly before this change (see diff to master). That results in `apt-key` to use gpg2 to fetch the key. And that indeterministically errored, probably due to IPv6 issues. Since we now plainly use `cURL` and then pass the already downloaded key to `apt-key` we circumvent the problem with the unresolvable keyserver alltogether.

Reply:
```suggestion
          # We used to call apt-key adv <keyserver> directly, which makes
          # apt-key use gpg2 to fetch the key.  That indeterministically
          # errored, probably due to IPv6 issues in CI. We now use cURL to
          # fetch the key, and pass the already downloaded key to apt-key. That
          # way, unresolvable keyservers are no longer an issue.
          curl -fsSL https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add -
```

Reply:
We would need to put that kind of comment above every place where we pull custom keys. That is two times alone in this workflow, more in the other workflows, Dockerfiles etc. Not using `gpg` / `apt-key` directly is a general learning and can go into our internal docs but there is no single right place to put that kind of comment in the codebase.

Reply:
Having it somewhere is better than having it nowhere at all. Where would you put it?